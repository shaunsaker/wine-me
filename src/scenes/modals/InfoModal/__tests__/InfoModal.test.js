import React from 'react';
import renderer from 'react-test-renderer';

import { InfoModal } from '..';

jest.mock('../../../../components/CodePushStatus', () => 'CodePushStatus');

// Fixes _bezier is not a function bug
jest.useFakeTimers();

describe('InfoModal', () => {
  describe('renders', () => {
    it('renders with minimum required props', () => {
      const component = renderer.create(<InfoModal />);

      expect(component).toMatchSnapshot();
    });
  });
});
