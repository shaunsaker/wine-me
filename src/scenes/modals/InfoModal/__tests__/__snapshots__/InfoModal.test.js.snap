// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoModal renders renders with minimum required props 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgba(0, 0, 0, 0.70)",
      "bottom": 0,
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
      "transform": Array [
        Object {
          "translateY": 1334,
        },
      ],
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
        "margin": 30,
      }
    }
  >
    <View
      style={
        Object {
          "alignSelf": "stretch",
          "backgroundColor": "#FFFFFF",
          "borderRadius": 5,
          "maxWidth": 680,
          "overflow": "hidden",
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#48466D",
            "borderColor": null,
            "borderWidth": 0,
            "elevation": 2,
            "justifyContent": "center",
            "paddingVertical": 10,
            "shadowColor": "#212121",
            "shadowOffset": Object {
              "height": 1,
              "width": 0,
            },
            "shadowOpacity": 0.17,
            "shadowRadius": 2,
          }
        }
      >
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
              "padding": 7.5,
              "position": "absolute",
              "right": 0,
            }
          }
          testID="lightbox.button.close"
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 12,
                },
                Object {
                  "color": "#FFFFFF",
                  "fontSize": 24,
                },
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            Óóç
          </Text>
        </View>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#FFFFFF",
              "fontFamily": "Nunito-SemiBold",
              "fontSize": 17,
            }
          }
        >
          Info
        </Text>
      </View>
      <View
        style={
          Object {
            "paddingHorizontal": 15,
            "paddingVertical": 20,
          }
        }
      >
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "center",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#212121",
                "fontFamily": "Nunito-Regular",
                "fontSize": 15,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "fontFamily": "Nunito-SemiBold",
                }
              }
            >
              Version:
               
            </Text>
            1.2.0 | 
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#212121",
                "fontFamily": "Nunito-Regular",
                "fontSize": 15,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "fontFamily": "Nunito-SemiBold",
                }
              }
            >
              Build:
               
            </Text>
            9 | 
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#212121",
                "fontFamily": "Nunito-Regular",
                "fontSize": 15,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "fontFamily": "Nunito-SemiBold",
                }
              }
            >
              Code:
               
            </Text>
            8
          </Text>
        </View>
        <View
          style={
            Object {
              "marginTop": 10,
            }
          }
        >
          <CodePushStatus />
        </View>
      </View>
    </View>
  </View>
</View>
`;
